<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Center</class>
 <widget class="QWidget" name="Admin_Center">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Admin_Center</string>
  </property>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>欢迎&lt;管理员&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUserInfo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>351</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnUserInfo
{
    color:white;
    background-color:rgb(31 , 174 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}
QPushButton#btnUserInfo:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}
QPushButton#btnUserInfo:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>用户个人信息管理</string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.people.profile.png</normaloff>:/imgIcon/appbar.people.profile.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnComment">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnComment
{
    color:white;
    background-color:rgb(105 , 27 , 184);
    border: 1px solid rgb(114 , 0 , 172);
}
QPushButton#btnComment:hover
{
    color:white;
    background-color:rgb(70 , 23 , 180);
    border: 1px solid rgb(11 , 137 , 234);
}
QPushButton#btnComment:pressed
{
    color:white;

    background-color:rgb(114 , 0 , 172);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>评语管理 </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.draw.pencil.reflection.png</normaloff>:/imgIcon/appbar.draw.pencil.reflection.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModifyPwd">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>411</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnModifyPwd
{
    color:white;
    background-color:rgb(0 , 30 , 78);
    border: 1px solid rgb(0 , 77 , 96);
}
QPushButton#btnModifyPwd:hover
{
    color:white;
    background-color:rgb(37 , 37 , 37);
    border: 1px solid rgb(11 , 137 , 234);
}
QPushButton#btnModifyPwd:pressed
{
    color:white;
    background-color:rgb(0 , 77 , 96);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>管理员密码修改</string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.cog.png</normaloff>:/imgIcon/appbar.cog.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTime">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnTime
{
    color:white;
    background-color:rgb(27 , 88 , 184);
    border: 1px solid rgb(0 , 106 , 193);
}
QPushButton#btnTime:hover
{
    color:white;
    background-color:rgb(0 , 106 , 193);
    border: 1px solid rgb(11 , 137 , 234);
}
QPushButton#btnTime:pressed
{
    color:white;
    background-color:rgb(37 , 115 , 236);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>时间设置</string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.calendar.year.png</normaloff>:/imgIcon/appbar.calendar.year.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnExit
{
    color:white;
    background-color:rgb(184 , 27 , 27);
    border: 1px solid rgb(174 , 17 , 61);
}

QPushButton#btnExit:hover
{
    color:white;
    background-color:rgb(184 , 27 , 108);
    border: 1px solid rgb(11 , 137 , 234);
}
QPushButton#btnExit:pressed
{
    color:white;
    background-color:rgb(176 , 30 , 0);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.png</normaloff>:/imgIcon/appbar.close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageIcon.qrc"/>
 </resources>
 <connections/>
</ui>
